staging/src/k8s.io/apimachinery/pkg/util/wait/wait.go

```diff
@@ -177,6 +177,32 @@ type Backoff struct {
 	Factor   float64       // Duration is multiplied by factor each iteration
 	Jitter   float64       // The amount of jitter applied each iteration
 	Steps    int           // Exit with error after this many steps
+	Cap      time.Duration // The maximum amount of time to wait if set
```

Comment:
is Jitter allowed to exceed Cap?

Reply:
In this, yes.  We wouldn’t want to line up again on cap.

Reply:
makes sense, just document that the cap is +/- jitter